comps/dataprep/redis/langchain/prepare_doc_redis.py

```diff
@@ -10,7 +10,7 @@
 
 # from pyspark import SparkConf, SparkContext
 import redis
-from config import EMBED_MODEL, INDEX_NAME, KEY_INDEX_NAME, REDIS_URL
+from config import EMBED_MODEL, INDEX_NAME, KEY_INDEX_NAME, REDIS_URL, SEARCH_BATCH_SIZE
 from fastapi import Body, File, Form, HTTPException, UploadFile
 from langchain.text_splitter import RecursiveCharacterTextSplitter
 from langchain_community.embeddings import HuggingFaceBgeEmbeddings, HuggingFaceHubEmbeddings
```

Comment:
should we replace HuggingFaceHubEmbeddings to HuggingFaceEndpointEmbeddings, refer this pr https://github.com/opea-project/GenAIComps/pull/417.